public/index.html

```diff
@@ -20,18 +21,209 @@
     -->
     <title>React App</title>
   </head>
-  <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+    <!--<div id="root"></div> -->
+   <body class="app">
+        <header class="header">
+            <img src="/images/dots.png"  width="27" height="5" />
```

Comment:
Папку `images `стоит положить в папку `public`
Также в пути стоит использовать %PUBLIC_URL%
```suggestion
            <img src="%PUBLIC_URL%/images/dots.png"  width="27" height="5" />
```

При сборке путь заменится на корректный

Это относится ко всем местам